packages/application-shell/src/track-performance.ts

```diff
@@ -8,7 +8,6 @@ const trackPerformance = () => {
   new Perfume({
     logging: false,
     dataConsumption: true,
-    largestContentfulPaint: true,
```

Comment:
This is apparently not present anymore in the options. I think it's just a default now.